paasta_tools/kubernetes_tools.py

```diff
@@ -237,19 +237,19 @@
 # For detail, https://github.com/kubernetes-client/python/issues/553
 # This hack should be removed when the issue got fixed.
 # This is no better way to work around rn.
-class MonkeyPatchAutoScalingConditions(V2beta2HorizontalPodAutoscalerStatus):
+class MonkeyPatchAutoScalingConditions(V2HorizontalPodAutoscalerStatus):
```

Comment:
i kinda wonder if we still need this - the linked GH issue fizzled out a while back,
but hopefully it's actually been fixed since we added this!

Reply:
Good call, I will take a look at this and remove if we can!